app/layout/community/published-projects/component.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { useEffect } from 'react';
+
+import { useDebouncedCallback } from 'use-debounce';
+import { useSelector, useDispatch } from 'react-redux';
+import { setSearch } from 'store/slices/community/projects';
+
+import Link from 'next/link';
+
+import { format } from 'd3';
+
+import Button from 'components/button';
+import Icon from 'components/icon';
+import Loading from 'components/loading';
+import Search from 'components/search';
+import Wrapper from 'layout/wrapper';
+
+import { usePublishedProjects } from 'hooks/projects';
+
+import ARROW_DOWN_SVG from 'svgs/ui/arrow-right-2.svg?sprite';
+import DOWNLOAD_SVG from 'svgs/ui/download.svg?sprite';
+
+export interface CommunityProjectsListProps {
+
+}
+
+export const CommunityProjectsList: React.FC<CommunityProjectsListProps> = () => {
+  const {
+    data: publishedProjectsData,
+    isFetching: publishedProjectsIsFetching,
+    isFetched: publishedProjectsIsFetched,
+  } = usePublishedProjects();
+
+  const { search } = useSelector((state) => state['/community/projects']);
```

Comment:
There is an error when I try to get to this page related to this. Did you fix it in this PR or in the other one?

Reply:
It will be on next PR (detail of published project)